tests/test_formatter.py

```diff
@@ -55,7 +55,7 @@ def setUp(self):
 
         self.security_error = Violation(
             code="S102",  # This is coming from flake8-bandit
-            filename="examples/insecure-code.py",
+            filename="./examples/insecure-code.py",
```

Comment:
Can you keep one of the `./` cases or introduce a new one so that the `./` is not required?

I wonder if the `./` has always / will always be there or whether that has changed at some point.

Reply:
Good idea! I added an extra test case to check for this.